tls/s2n_prf.c

```diff
@@ -500,6 +500,9 @@ static int s2n_prf(struct s2n_connection *conn, struct s2n_blob *secret, struct
 
 int s2n_tls_prf_master_secret(struct s2n_connection *conn, struct s2n_blob *premaster_secret)
 {
+    POSIX_ENSURE_REF(conn);
+    POSIX_ENSURE_REF(premaster_secret);
+
```

Comment:
These initial ENSURE_REF checks are intended to defend against segfaults due to invalid memory accesses. Is it possible for a NULL premaster_secret to cause a segfault in this method?